tests/cross_functional/kcs/test_selinux_relabel_solution.py

```diff
@@ -300,15 +304,32 @@ def test_selinux_relabel_for_existing_pvc(
         self.apply_selinux_solution_on_existing_pvc(self.pvc_obj)
 
         # Delete pod so that fix will be applied for new pod
+        POD_OBJ = ocp.OCP(
+            kind=constants.POD, namespace=config.ENV_DATA["cluster_namespace"]
+        )
+        assert modify_deploymentconfig_replica_count(
+            deploymentconfig_name=self.pod_obj.get_labels().get("name"), replica_count=0
+        ), "Failed to scale down deploymentconfig to 0"
+        POD_OBJ.wait_for_delete(resource_name=self.pod_obj.name, timeout=600)
```

Comment:
shouldn't it be pod_obj.wait_for_delete(resource_name=self.pod_obj.name, timeout=600) ?